components/datadog/agent/host_windowsos.go

```diff
@@ -115,7 +115,7 @@ func getAgentURL(version agentparams.PackageVersion) (string, error) {
 	fullVersion := fmt.Sprintf("%v.%v", version.Major, minor)
 
 	if version.PipelineID != "" {
-		return getAgentURLFromPipelineID(version.PipelineID)
+		return getAgentURLFromPipelineID(version.PipelineID, version.Major)
```

Comment:
We should not I think since the `7` was hardcoded [here](https://github.com/DataDog/test-infra-definitions/pull/1192/files#diff-452201b235802238a1b82b1a5dda9bb00e71c9922104eaffd6fcab993c150475R165)

Reply:
Updated!